README.md

```diff
@@ -104,16 +107,9 @@ AWS-TUI | Terminal User Interface for AWS | ![Badge](https://img.shields.io/badg
 <br />
 
 ### You can find my blogs on [Medium](https://ankush-chavan.medium.com/)
+
 - [Running GUI Applications in Docker Container](https://ankush-chavan.medium.com/running-gui-applications-in-docker-container-f088e48dab9b)
 - [Mobile Terminal using Flutter, Firestore and StreamBuilder](https://ankush-chavan.medium.com/mobile-terminal-using-flutter-firestore-and-streambuilder-dd80424f3737)
 - [Math-Driven Approach towards Animation in Flutter](https://ankush-chavan.medium.com/math-driven-approach-towards-animation-in-flutter-4936daf1af2b)
 - [Getting started with Redhat OpenShift Container Platform(RHOCP)](https://ankush-chavan.medium.com/getting-started-with-redhat-openshift-container-platform-rhocp-34520e133608)
 - [Sending CLI Output to Whatsapp using Command Line](https://ankush-chavan.medium.com/sending-cli-output-to-whatsapp-using-command-line-e2213b6c1a8b)
-
-[linkedin]: https://www.linkedin.com/in/ankushchavan/
```

Comment:
The links can be stored separately. So, that If any of these link changes in future, the changes can be made at here. Also, it looks much cleaner, according to me.

Reply:
> The links can be stored separately. So, that If any of these links changes in future, the changes can be made here. Also, it looks much cleaner, according to me.

I removed that to align the buttons to the center. If we keep that, they cannot be aligned to the center, they will be kept to the left and IMO it looks bad.

Still, if you want it back, I can add it back if you want. 🙂

Reply:
Looks good. All the buttons/badges should be in a single line if the screen is big

Reply:
> Looks good. All the buttons/badges should be in a single line if the screen is big

Ah, I intentionally made it like that so that it doesn't look in a single line. IMO, it looks better.